config/metadata/rulesMeta.yaml

```diff
@@ -495,31 +506,31 @@ groups:
               - EMAThroughputSampler
               - WindowedThroughputSampler
               - TotalThroughputSampler
-        summary: is the dynamic sampler to use if the rule matches.
+        summary: is the Dynamic Sampler to use if the rule matches.
         description: >
           The sampler to use if the rule matches. If this is set, the sample
           rate will be determined by this downstream sampler. If this is not set,
-          the sample rate will be determined by the Drop flag or the SampleRate
-          field.
+          the sample rate will be determined by the `Drop` flag or the 
+          `SampleRate` field.
       - name: Drop
         type: bool
         summary: indicates whether to drop the trace.
         description: >
-          Indicates whether to drop the trace if it matches this rule. If true,
-          the trace will be dropped. If false, the trace will be kept.
+          Indicates whether to drop the trace if it matches this rule. If `true`,
+          then the trace will be dropped. If `false`, then the trace will be kept.
       - name: SampleRate
         type: int
         summary: is the sample rate to use.
         description: >
-          If the rule is matched, there is no Sampler specified, and the Drop
-          flag is false, then this is the sample rate to use.
+          If the rule is matched, there is no Sampler specified, and the `Drop`
+          flag is `false`, then this is the sample rate to use.
       - name: Conditions
         type: objectarray
         summary: is the list of conditions to use to determine whether the rule matches.
         description: >
           Conditions is a list of conditions to use to determine whether the
           rule matches. All conditions must be met for the rule to match. If
-          there are no conditions, the rule will always match (this is typically
+          there are no conditions, then the rule will always match (this is typically
```

Comment:
@kentquirk Asking for clarity - What is "this" supposed to be in the sentence? The rule or the condition?

Reply:
Maybe remove parens and say "A no-condition rule is typically used for the last rule to provide a default behavior."

Reply:
Sounds good! Adding now.